### PR TITLE
Fixes cookbook for fixers

### PR DESCRIPTION
**Pull request by greeflas**
- Updated file header and array syntax
- Fixed code style in code samples

### CODE REVIEW

**Comment by ntzm** [comment]
See #3118

**Comment by greeflas** [comment]
@ntzm but what about other fixes? :)

**Comment by ntzm** [comment]
What other fixes? They are all code style related

**Comment by greeflas** [comment]
@ntzm PR #3118 has only indentation fixes. This PR is updates array syntax, file header and adds additional PHPDoc blocks.

**Comment by greeflas** [comment]
@dmvdbrugge ok. 
1. For me as for developer of fixers would be good to see updated doc with updated syntax.
2. Fixer will not add additional PHPDoc that was added in this PR.

**Comment by SpacePossum** [comment]
I know the code should be fixed after running the tool, but as a reader being told to do on thing and than have it changed directly after by a tool feels odd.

If was only on sample-code that is to be fix later by the rule of the cookbook than I get it, but having the skeleton code in shape is still up to date is still a good thing IMHO.

File: doc/COOKBOOK-FIXERS.md [hunk 1]
```diff
@@ -113,10 +115,15 @@ use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
  * @author Your name <your@email.com>
  *
  * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\Comment\RemoveCommentsFixer
```
Comment (keradus) [inline doc/COOKBOOK-FIXERS.md]: :+1:

**Comment by keradus** [comment]
Thank you @greeflas.

**Pull request merged**